src/strategy/indicators/cmf.ts

```diff
@@ -0,0 +1,30 @@
+// Chaikin Money Flow
```

Comment:
If you start the comment with `/**` and end it with `*/` afterwards (multi line comment) an ide like vscode will display the comment as class comment in the tooltip. 
i would suggest to add the comment below as the class comment